### PR TITLE
Adding bootstrap instructions

### PR DESCRIPTION
**Pull request by davidschou**
This can help new users get up and going quickly

### CODE REVIEW

**Comment by ZedPea** [comment]
Could we potentially move this into the TurtleCoind / simplewallet issues section? Next to the "Can I skip the syncing section" would be good imo, as they're quite similar. Might need to reword that section so the difference is more obvious, or potentially merge them. 

If you want it to be in the other section that's fine by me, but I'd like for the "I have a question which wasn't answered here, what should I do" bit to be last, as it seems the natural conclusion to the FAQ.

I would also suggest hosting the help.txt on this repo with an appropriate name, but that's completely up to you of course. I have push access to this repo so can approve that, but I also have a pull request open to add the FAQ to the wiki, so you'd have to open a pull request there to host it so the link would work, if you decided to host it here.

I wrote that kinda badly, hope you understood what I meant.

Feel free to disagree, I'm not anyone of importance, I just wrote the FAQ so I got given push access so I can update it with more ease.

**Comment by davidschou** [comment]
I like the suggestion about having the help.txt on this repo because it would be super useful for folks to see instructions here rather than clicking to the text document.   What do you think about:

1.  I created a file here docs/"How to bootstrap the blockchain.md" with the help.txt instructions 
2.  Update the verbiage with: 

```
* **Q: Can I speed up the syncing of the blockchain?*** 

    A: You can grab a recent copy of the blockchain and incrementally sync from there.  See "How to bootstrap the blockchain.md" for instructions.
```

3.  Put my question under "Can I skip the syncing?" (as you suggested)

**Comment by ZedPea** [comment]
Yep, that sounds great to me. You can also use a relative link so if someone forks the repo or whatever, the link will still work to your file: https://github.com/blog/1395-relative-links-in-markup-files
Not 100% sure if links will work fine with spaces, but that's easily tested.

File: FAQ.md [hunk 1]
```diff
@@ -52,14 +52,14 @@
...
     A: Yes, in case you get banned, or a pool goes down for sometime, you can keep mining. A good amount to have would be at least three.
-    
+
 * **Q: Where can I find a list of pools?**
 
     A: http://turtle-coin.com/#pools - This website also shows some other nice stats like hashrate, and min payout.
 
 * **Q: I get a socket error when connecting to a pool. What am I doing wrong?**
 
-    A: Generally, this is due to an incorrectly configured config. In the directory where your xmr-stak.exe is, there should be a file called `config.txt`. Open this up, and check that it looks similar to this:
```
Comment (davidschou) [inline FAQ.md]: looks like Atom did this automatically

File: FAQ.md [hunk 2]
```diff
@@ -52,14 +52,14 @@
 * **Q: Should I set up multiple pools?**
 
     A: Yes, in case you get banned, or a pool goes down for sometime, you can keep mining. A good amount to have would be at least three.
-
```
Comment (davidschou) [inline FAQ.md]: looks like Atom did this automatically

**Comment by davidschou** [comment]
@ZedPea this is ready to review when you have a moment.  Let me know how it looks now, thanks!

**Comment by ZedPea** [comment]
Looks great, thank you!

**Pull request merged**